### ISSUE TITLE
Av1an leaves python processes open

### ISSUE THREAD

**Issue by n9Mtq4** [master-of-zen/Av1an#95]
I noticed this issue on Manjaro linux when av1an crashes. I had many python processes open from old av1an commands. This definitely happens when av1an crashes and might happen when it exits normally.

To reproduce, try to start an encoding that will crash (like the following command). After the crash, there will still be python processes in a task manager.

```
python ./av1an.py -i source.mkv -enc aom -ff ' --invalid option ' --split_method aom_keyframes
```

Can other people reproduce this?

This seems to be fixed by replacing the `os.kill` in the terminate function with `sys.exit(1)`. Is there a reason that `os.kill` is used over `sys.exit(1)`?

**Comment by Quantum1248** [comment]
I use Manjaro and can reproduce the issue.

**Comment by luigi311** [comment]
Can confirm. Had a bunch of python process open on clearlinux

**Comment by luigi311** [comment]
This happens when even when i ctrl+c av1an to kill it. It will still leave a thread hanging and i have to kill it via htop

**Comment by n9Mtq4** [comment]
This was fixed in d1b44bb2364c3b8076abdf1358b156f5e3f8e689.
